src/components/Population.js

```diff
@@ -6,36 +6,6 @@ class Population extends Cache {
   constructor() {
     super();
   }
-  //   fetch("https://restcountries.eu/rest/v2/")
```

Comment:
Please remove these commented out lines of code (gordon showing harun something)